.env.default

```diff
@@ -11,7 +11,8 @@ ADMIN_MAIL=admin@example.com
 PROJECT_INSTALL=
 IMAGE_PHP=skilldlabs/php:73-fpm
 IMAGE_NGINX=skilldlabs/nginx:1.16
-IMAGE_FRONT=skilldlabs/frontend:zen
+IMAGE_FRONT=node:lts-alpine
```

Comment:
That is what I'm totally against - node just released new lts and who sure that all our scripts compatible? And packages from from yarn.lock...

Reply:
```
iberdinsky
sure let's define it as fixed version

apostnikov
It also no-go because then will need to upgrade each project instead of one custom image

iberdinsky
in case of one custom image we will have to do much more actions that simple node:lts -> node:XX

apostnikov
Why? It works for our PHP images for years

iberdinsky
you mean packageless image which just import node with version?

apostnikov
At least yes + I'd packaged common stack (webpack, postcss)
```

Reply:
Created follow-up at https://github.com/skilld-labs/skilld-docker-container/issues/179

Reply:
I think we need to let `IMAGE_FRONT=node:lts-alpine` in `.env` file but doesn't allow to install project in case `IMAGE_FRONT=node:lts-alpine` with message like "You requested `latest` version of nodejs image, please change it with proper version to really protect version of nodejs during development process." For each project, this version must be individual and you will need to privde something like `IMAGE_FRONT=node:12.13.1-alpine3.10` for each project then it starts.